### ISSUE TITLE
Why NRE exceptions are being thrown after the 19400 KX13 build?

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#74]
### Brief bug description

After the 19400 build had been created (Beta4), there've been numerous NRE exceptions all over the place during each home page request.

### Repro steps

1. Run the home page request in the debugger.
2. Inspect the Diagnostic tools pane -> Events tab.

### Expected behavior

What is the correct behavior?

### Test environment

 - Platform/OS: [e.g. .NET Core 3.1, iOS, Windows]
 - Browser [e.g. Chrome, Safari, Edge Chromium, Edge]
 - Version [e.g. 81]

### Additional context

Add any other context about the problem here. If you found an issue where a similar bug has been reported, add a link here.

### Screenshots

Add links to screenshots, if you deem it appropriate.

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`
